code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -1203,6 +1206,10 @@
 	update_icon()
 
 /obj/item/hierophant_club/ui_action_click(mob/user, action)
+	var/turf/user_turf = get_turf(user)
+	if(!SSmapping.level_trait(user_turf, ZTRAIT_MINING))
+		to_chat(user, "<span class='warning'>The club fizzles wekaly, it seem his power does not reach to this realm.</span>")
+		return
```

Comment:
I don't see why this is needed, the teleport ability is a unique one and doesn't seem too overpowered for a big ass club that you can't backpack

Reply:
fully agree, I talked to miners about this, will be fixed

Reply:
let people use the clubs teleport function on station